projects/core/src/routing/configurable-routes/configurable-routes.service.ts

```diff
@@ -125,26 +122,26 @@ export class ConfigurableRoutesService {
   private translateRoutePath(
     route: Route,
     routesTranslations: RoutesTranslations
-  ): Route[] {
-    return this.getTranslatedPaths(route, 'cxPath', routesTranslations).map(
-      translatedPath => {
-        return { ...route, path: translatedPath };
-      }
-    );
+  ): Route {
+    const paths = this.getTranslatedPaths(route, 'cxPath', routesTranslations);
+    const matcher = paths.length
```

Comment:
I would really love to use our `custom/default/copied` matched logic only if necessary (to mitigate eventual misalignments with possibly evolving built in one in the future). 
What about just simply using plain `path` if `paths.length == 1`? It could be also helpful for some route debugging (it is easier to reason from path string than from a function in runtime config).